Formula/timg.rb

```diff
@@ -22,6 +22,7 @@ class Timg < Formula
   depends_on "ffmpeg"
   depends_on "graphicsmagick"
   depends_on "jpeg-turbo"
+  depends_on "libdeflate"
```

Comment:
Why?

Reply:
timg is adding libdeflate in HEAD with [this commit](https://github.com/hzeller/timg/commit/818e47cdc16493bbb04fb88e2a33e377c95c132d)

Reply:
Then we should restrict this dependency to `head` until it lands in a stable release. There's no reason to force everyone to install `libdeflate` as a dependency while it's unused.